scripts/run_bazel_tests.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Usage: ./run_bazel_tests.sh
+
+# This runs the bazel test commands, ex:
+# bazel test //tests/Integration:asset && \
+# bazel test //tests/Integration:compute_small
+# etc
+
+echo "Running bazel tests"
+
+bazel run //tests/Integration:asset_update && \
```

Comment:
```suggestion
bazel test //tests/Integration:asset && \
```

Same goes for all other targets in **this** file.

Reply:
Not quite. `bazel test` just runs the test cases. `bazel run ...:{api}_update` will actually update the golden files for the integration tests but not run a test. These are two different things.

Reply:
Oops. I see my mistake now. Thanks for pointing it out.